src/core/CoreDatabaseManager.jl

```diff
@@ -5,18 +5,44 @@ mutable struct CoreDatabaseManager <: AbstractCoreDatabaseManager
 end
 
 function get_database(client::T, name::String)::CoreDatabase where {T<:AbstractCoreClient}
```

Comment:
Again, eliminate the where clause, also, no need to type name
(at most, put AbstractString, so that other things like SubStrings, ShortStrings, Strs, can work)